tests/test_helpers.py

```diff
@@ -17,6 +17,10 @@
       "--date", "2020-04-29", "-c", "afriend", "-d", "more work",
       "-m", "ascopatz", "-g", "SymPy-1.1"],
      "20ly_newprojectum has been added in projecta\n"),
+    (["helper", "a_proposal", "a new proposal", 500.0, "Kurt Godel", "USD"
```

Comment:
maybe make 2 or 3 tests that test reasonable edge cases (but don't go crazy to test every tiny possibility in this case)